xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources_fr.properties

```diff
@@ -3903,16 +3903,16 @@ platform.extension.distributionWizard.extension.outdatedextensionsStepTitle=Exte
 platform.extension.distributionWizard.extension.outdatedextensionsStepSummary=Mettre \u00E0 jour les extensions install\u00E9es
 platform.extension.distributionWizard.extensionsStepDescription=Les extensions fournissent des fonctionnalit\u00E9s suppl\u00E9mentaires en dessus du moteur d''ex\u00E9cution de XWiki. Ceux-ci sont d''habitude distribu\u00E9es en XAR (par exemple des {0}Applications XWiki{1}, des {2}macros wiki{3}, des {4}th\u00E8mes de couleur{5}) et en JAR (du code serveur qui inclut des {6}composants{7} et des {8}services{9}).
 
-### Missing: platform.extension.distributionWizard.extension.cleanStepTitle=Orphaned extensions
-### Missing: platform.extension.distributionWizard.extension.cleanStepSummary=Make sure orphaned extension dependencies are either removed or made top level.
-### Missing: platform.extension.distributionWizard.extension.cleanStep.noOrphaned=No orphaned dependency could be found in that instance.
-### Missing: platform.extension.distributionWizard.extension.cleanStep.orphaned=The following extensions have been installed as dependencies and are no longer required. You can either uninstall them (checked) or make them top level extensions if you still need them (unchecked).
-### Missing: platform.extension.distributionWizard.extension.cleanStep.button.cleanapply=Continue
-### Missing: platform.extension.distributionWizard.extension.cleanStep.button.cleanapplyreport=Continue
-### Missing: platform.extension.distributionWizard.extension.cleanStep.button.back=Back
-### Missing: platform.extension.distributionWizard.extension.cleanStep.apply.title=Apply
-### Missing: platform.extension.distributionWizard.extension.cleanStep.report.uninstalled=The following extensions have been uninstalled:
-### Missing: platform.extension.distributionWizard.extension.cleanStep.report.top=The following extensions have been made top level:
+platform.extension.distributionWizard.extension.cleanStepTitle=Extensions orphelines
```

Comment:
@tmortagne should be translated differently? I saw that orphaned dependencies became orphaned extensions?

Reply:
Yes it should.

Reply:
Ok, waiting the change before merging then.